server/routes/config/authorisation.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
   '/dashboard(Initial|Recat)(\\?.+)?': { authorised: ['ROLE_APPROVE_CATEGORISATION'] },
   '/categoryHistory/\\d+': { authorised: ['BOOKING_ID_IN_CASELOAD'] },
   '/tasklist/\\d+(\\?.+)?': { authorised: ['ROLE_CREATE_CATEGORISATION'] },
-  '/tasklist/supervisor/(outcome/)?\\d+': { authorised: ['ROLE_APPROVE_CATEGORISATION'] },
+  '/tasklist/supervisor/(outcome/)?\\d+.+': { authorised: ['ROLE_APPROVE_CATEGORISATION'] },
```

Comment:
why .+

Reply:
to accept additional parameters, e.g. catType=INITIAL